include/onnxruntime/core/providers/coreml/coreml_provider_factory.h

```diff
@@ -31,9 +31,14 @@ enum COREMLFlags {
   // Create an MLProgram. By default it will create a NeuralNetwork model. Requires Core ML 5 or later.
   COREML_FLAG_CREATE_MLPROGRAM = 0x010,
 
+  // Exclude ANE as sometimes this decrease performance
+  // https://developer.apple.com/documentation/coreml/mlcomputeunits?language=objc
+  // there are four compute units:
+  // MLComputeUnitsCPUAndNeuralEngine|MLComputeUnitsCPUAndGPU|MLComputeUnitsCPUOnly|MLComputeUnitsAll
+  COREML_FLAG_USE_CPU_AND_GPU = 0x020,
```

Comment:
if we're calling it `USE_CPU_AND_GPU` and not calling it `EXCLUDE_ANE`, I'd suggest documenting it as such. i.e., something like "use CPU and GPU for computation in CoreML". you can still mention that using this to exclude the ANE may increase performance.

Reply:
For different models, running on different hardwares, the performance varies. It's hard to say which option is the best. To try with different CU and pick the best one which shows better performance is a good way.

Reply:
sure, you can document the performance implications however you see fit.

what I'm trying to suggest is to be consistent with the name, "COREML_FLAG_USE_CPU_AND_GPU" and the documentation for it, "Exclude ANE". I think either changing the name to "COREML_FLAG_EXCLUDE_ANE" or changing the documentation to "Use CPU and GPU" would make it more consistent.

Reply:
sounds good. Will fix it in the next PR.